test/testFiles/multiLineCommentEven.txt

```diff
@@ -0,0 +1 @@
+comments ["this is a", "multiline comment"]
```

Comment:
Could we add another test for multi-line comments with new lines within the array:
comments [
"this is a",
"multiline comment"
]
Would be good to cover this case! Thank you 🙏🏻

Reply:
done!